### ISSUE TITLE
long long press

### ISSUE THREAD

**Issue by GoogleCodeExporter** [klausw/hackerskeyboard#414]
```
I suggest a "long long" press.

Such feaure so suggested:
1. Is activated when while enabled after the duration of the "long press", the 
key remains pressed for the additional duration of the "long long press".
2. When such "long long pres" is enabled, such "long press" is modified such 
that only the glyph as modified by the shift modifier shall be available for 
such "long press".
Example, a long press of "e" shall result "E", but not "é" nor "ë".
3. Such long long press shall have available to it, such uncommon modifiers as 
"é" "★" etc.


Rationale:

1. It is much more common the case for a typist to require the shift modifier 
rather than any extended keyset.
2. It is found by the submitter to be sufficiently easy to accidentally enter a 
glyph of the unicode extended set as to cause significant anoyance.

tl;dr:
Instead of a long press, have a "long" and "very long" press. The long pres has 
shift modifier semantics, and very long shall have the full shebang.


```

Original issue reported on code.google.com by `r.p.nich...@gmail.com` on 30 Apr 2014 at 5:12
